.travis.yml

```diff
@@ -34,38 +45,48 @@ jobs:
         - tox -e $TOX_ENV
 
     - <<: *default
+      arch: amd64
```

Comment:
redundant (here and in all the subsequent blocks)

Reply:
Addressed the comments.